client/src/App.scss

```diff
@@ -0,0 +1,89 @@
+%screen-centering{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App{
+  height: 100vh;
+  display: flex;
+  color: white;
+}
+
+.App_aside{
+  width: 50%;
+  @extend %screen-centering;
+  // display: flex;
+  // justify-content: center;
+  // align-items: center; 
```

Comment:
Let's try to avoid commented out code like this - in the rest of the PR as well - the idea behind git is so we don't have to do this